### PR TITLE
Empty state

### PR DESCRIPTION
**Pull request by shiranmeir**
1. Empty state for:

- Course editor screen
- Course items [isCourseItems =false]
- Items [isFilteredCourseItems=false]
- Courses table [replace 'tableData' with '[]' ]
- Users started/completed course [check with props isEmpty = true]
- Charts [in PieBarSummary component check with value=0]
- Line chart (with mock data)[check with props isEmpty = true]
- Quiz score char [check with 'hasData =false']

2. Empty state playground
3. New empty state icons 
4. New add button component

*missing (state not exist yet): 
-Empty lesson
-Search results for outline- no results

### CODE REVIEW

File: src/components/Screen/Playground/components/Empty.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import WMEmpty from '../../../common/WMEmpty';
+import Icon, { IconType } from '../../../common/Icon';
+import AddButton from '../../../common/buttons/AddButton';
+import CourseSummaryChart from '../../../common/AnalyticsCharts/CourseSummaryChart';
+
+export default function Counter(): ReactElement {
+  return (
+    <>
+      <WMEmpty
+        description="Start building your course by creating lessons and draging items from the Items List"
```
Comment (RonnyO) [inline src/components/Screen/Playground/components/Empty.tsx:13]: dra**g**ging, not draging. I know the mistake was in the design, we should never trust the designer with copy 😁

File: src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx [hunk 2]
```diff
@@ -47,15 +47,32 @@ export default function CourseSummaryChart({ summaryData }: ICourseSummaryChart)
     });
   };
 
+  const usersStarted = 20;
+  const usersCompleted = 0;
+
+  const isEmpty = summaryData.data.days === [] ? true : false;
```
Comment (RonnyO) [inline src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx:53]: Lose the ternary. It's already a boolean after the comparison.

Comment (RonnyO) [reply src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx:53]: Wait, the entire condition is broken, as `[] === []` is `false`. Please re-check this actually works

File: src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx [hunk 3]
```diff
@@ -13,21 +13,25 @@ export default function QuizScoreChart({ quizScoreData }: IQuizScoreData): React
...
+
   return (
     <WMCard title={title} className={classes['course-average']}>
       <div className={classes['course-average-content']}>
         <WMProgress
           className={classes['course-average-chart']}
           percent={average}
           type={ProgressType.Circle}
-          format={() => average}
+          format={hasData ? () => average : () => '- -'}
```
Comment (RonnyO) [inline src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx:25]: Why not do the ternary inside the function?

File: src/components/common/WMEmpty/index.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement, ReactNode } from 'react';
+import { Empty } from 'antd';
+import { EmptyProps } from 'antd/lib/empty';
+import cc from 'classcat';
+
+import classes from './style.module.scss';
+
+export interface IWMEmpty extends EmptyProps {
+  children?: ReactNode;
```
Comment (RonnyO) [inline src/components/common/WMEmpty/index.tsx:9]: It's already inside `EmptyProps`, just use that if you don't add anything

File: src/components/Screen/Playground/components/Empty.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      </WMEmpty>
+      <Divider />
+      <WMEmpty
+        description={
+          <>
+            <div>There are no items available.</div>
+            <div>Create Walk-thrus and resources from the WalkMe Editor.</div>
+          </>
+        }
+        image={null}
```
[the remark this answers is not in the document]
Comment (RonnyO) [reply src/components/Screen/Playground/components/Empty.tsx:26]: Oh I see `Empty` has a default image. But in 90% of the `WMEmpty` usages you pass `null`, so maybe it makes sense to change the default to `null`?

File: src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx [hunk 6]
```diff
@@ -47,15 +47,32 @@ export default function CourseSummaryChart({ summaryData }: ICourseSummaryChart)
...
+            isEmpty={isEmpty}
+          >
+            <LegendContent number={usersStarted} description="52% of users with TeachMe access" />
           </WMLegend>
-          <WMLegend title="User Completed" dotStatusColor="#8812FF">
-            <LegendContent number={2130} description="47% of users who started courses" />
+          <WMLegend
+            usersStarted={usersStarted}
+            usersCompleted={usersCompleted}
+            title="User Completed"
```
Comment (RonnyO) [inline src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx:71]: _User**s**_ in both cases. I know this isn't new but it should be correct.

File: src/components/common/WMLegend/index.tsx [hunk 7]
```diff
@@ -5,33 +5,44 @@ import Header from '../Header';
...
 import classes from './style.module.scss';
+import EmptyLegend from './EmptyLegend';
 
 export interface IWMLegend {
   className?: string;
   title: string;
   dotStatusColor: string;
   children?: ReactNode;
+  usersStarted?: number;
+  usersCompleted?: number;
```
Comment (RonnyO) [inline src/components/common/WMLegend/index.tsx:16]: Why? You don't use them

File: src/components/common/buttons/AddButton/index.tsx [hunk 8]
```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement } from 'react';
+import cc from 'classcat';
+
+import { PlusOutlined } from '@ant-design/icons';
```
Comment (RonnyO) [inline src/components/common/buttons/AddButton/index.tsx:4]: The empty line should go below this one rather than above it.

File: src/styles/variables.scss [hunk 9]
```diff
@@ -53,6 +54,8 @@
   --neutral200: #b3b3b3;
   --neutral100: #e1e9ee;
 
+  --emptytext: rgba(0, 0, 0, 0.5);
```
Comment (RonnyO) [inline src/styles/variables.scss:57]: Add a dash please

File: src/styles/variables.scss [hunk 10]
```diff
@@ -82,4 +85,8 @@
 
   // switch
   --switch-off: #9ba1b0;
+
+  //disables
```
Comment (RonnyO) [inline src/styles/variables.scss:89]: `// disabled`

File: src/components/Screen/CourseEditorScreen/style.module.scss [hunk 11]
```diff
@@ -52,6 +52,19 @@
     .filters-bar {
       padding: 0 20px 13px;
     }
+
+    .items-container {
+      &.empty-state-items {
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CourseEditorScreen/style.module.scss:57]: I removed 'items-containe' class from css (.items-container can't get the same style as .empty-state-items)

File: src/components/Screen/CoursesScreen/index.tsx [hunk 12]
```diff
@@ -106,6 +106,22 @@ export default function CoursesScreen(): ReactElement {
...
+      <span>No courses yet</span>
+      <p>Start creating courses by clicking the button below</p>
+      <WMButton
+        className={classes['create-btn']}
+        shape="round"
+        variant={ButtonVariantEnum.Create}
+        icon={<PlusOutlined />}
+      >
+        Create Course
+      </WMButton>
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CoursesScreen/index.tsx:121]: The design changed after i wrote it, i asked the team about it and been told to leave it for now.

File: src/components/Screen/CoursesScreen/index.tsx [hunk 13]
```diff
@@ -106,6 +106,22 @@ export default function CoursesScreen(): ReactElement {
 
   const hasSelected = !!selectedRowKeys.length;
 
+  const customizeRenderEmpty = () => (
+    <div className={classes['empty-state']}>
+      <Icon className={classes['empty-icon']} type={IconType.CourseEmpty} />
+      <span>No courses yet</span>
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CoursesScreen/index.tsx:112]: Changed it to <h1>

File: src/components/Screen/Playground/components/Empty.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/Playground/components/Empty.tsx:26]: Right, i fixed it.

File: src/components/Screen/Playground/components/Empty.tsx (hunk 1 above)
Comment (shiranmeir) [reply src/components/Screen/Playground/components/Empty.tsx:13]: Oops....fixed

File: src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx (hunk 2 above)
Comment (shiranmeir) [reply src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx:53]: Changed it to hasData and fixed the props.

File: src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx (hunk 3 above)
Comment (shiranmeir) [reply src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx:25]: Ok changed it

File: src/components/common/WMEmpty/index.tsx (hunk 4 above)
Comment (shiranmeir) [reply src/components/common/WMEmpty/index.tsx:9]: Ok

File: src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx (hunk 6 above)
Comment (shiranmeir) [reply src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx:71]: OK

File: src/components/common/WMLegend/index.tsx (hunk 7 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/index.tsx:16]: Removed

File: src/components/common/buttons/AddButton/index.tsx (hunk 8 above)
Comment (shiranmeir) [reply src/components/common/buttons/AddButton/index.tsx:4]: OK

File: src/styles/variables.scss (hunk 9 above)
Comment (shiranmeir) [reply src/styles/variables.scss:57]: OK

File: src/styles/variables.scss (hunk 10 above)
Comment (shiranmeir) [reply src/styles/variables.scss:89]: OK

File: src/components/common/WMEmpty/index.tsx [hunk 14]
```diff
@@ -0,0 +1,33 @@
+import React, { ReactElement, ReactNode } from 'react';
+import { Empty } from 'antd';
+import { EmptyProps } from 'antd/lib/empty';
+import cc from 'classcat';
+
+import classes from './style.module.scss';
+
+export interface IWMEmpty extends EmptyProps {
+  image?: ReactNode | null;
+}
```
Comment (dvira) [inline src/components/common/WMEmpty/index.tsx:10]: There is no need to for `IWMEmpty`, image already exists in `EmptyProps`.

File: src/components/common/WMLegend/EmptyLegend.tsx [hunk 15]
```diff
@@ -0,0 +1,27 @@
...
+import cc from 'classcat';
+
+import Header from '../Header';
+import StatusDot, { DotType } from '../StatusDot';
+
+import classes from './style.module.scss';
+
+export interface IWMEmptyLegend {
+  title: string;
+}
```
Comment (dvira) [inline src/components/common/WMLegend/EmptyLegend.tsx:11]: Please add a line break here.

File: src/components/common/WMLegend/EmptyLegend.tsx [hunk 16]
```diff
@@ -0,0 +1,27 @@
...
+export default function EmptyLegend({ title }: IWMEmptyLegend): ReactElement {
+  return (
+    <>
```
Comment (dvira) [inline src/components/common/WMLegend/EmptyLegend.tsx:14]: No need for fragment.

File: src/components/common/WMLegend/EmptyLegend.tsx [hunk 17]
```diff
@@ -0,0 +1,27 @@
...
+      <div className={cc([classes['legend']])}>
```
Comment (dvira) [inline src/components/common/WMLegend/EmptyLegend.tsx:15]: No need for 'cc' here, plus I believe this className should be 'empty-legend'.

File: src/components/common/WMLegend/EmptyLegend.tsx [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+        <Header className={classes['legend-header']}>
+          <>
```
Comment (dvira) [inline src/components/common/WMLegend/EmptyLegend.tsx:17]: No need for fragment here also.

File: src/components/common/WMLegend/EmptyLegend.tsx [hunk 19]
```diff
@@ -0,0 +1,27 @@
...
+            <StatusDot className={classes['legend-dot-status']} type={DotType.Disable} />
+            <span className={classes['legend-title']}>{title}</span>
+          </>
+        </Header>
+        <span className={classes['empty-number']}>- -</span>
+        <span className={classes['legend-description']}> </span>
```
Comment (dvira) [inline src/components/common/WMLegend/EmptyLegend.tsx:23]: Do you really need this span? We normally don't add spans with white-space in them. What's the purpose?
If you need it but added the white-space so eslint won't give you a warning, simply make this span a self-closing element.

File: src/components/common/WMLegend/index.tsx [hunk 20]
```diff
@@ -5,33 +5,42 @@ import Header from '../Header';
 import StatusDot, { DotType } from '../StatusDot';
 
 import classes from './style.module.scss';
+import EmptyLegend from './EmptyLegend';
```
Comment (dvira) [inline src/components/common/WMLegend/index.tsx:8]: Move this before 'classes' please.

File: src/components/common/WMLegend/index.tsx [hunk 21]
```diff
@@ -5,33 +5,42 @@ import Header from '../Header';
...
+  hasData?: boolean;
 }
 
 export default function WMLegend({
   className,
   title,
   dotStatusColor,
   children,
+  hasData,
 }: IWMLegend): JSX.Element {
```
Comment (dvira) [inline src/components/common/WMLegend/index.tsx:24]: Replace 'JSX.Element' with 'ReactElement'.

File: src/components/common/WMLegend/index.tsx [hunk 22]
```diff
@@ -5,33 +5,42 @@ import Header from '../Header';
...
-            className={classes['legend-dot-status']}
-            type={DotType.Custom}
-            dotColor={dotStatusColor}
-          />
-          <span className={classes['legend-title']}>{title}</span>
-        </>
-      </Header>
-      {children}
-    </div>
+    <>
```
Comment (dvira) [inline src/components/common/WMLegend/index.tsx:26]: I assume you wrapped this component's return value with fragment because the ternary prompts an eslint warning. Right?

File: src/components/common/WMLegend/index.tsx [hunk 23]
```diff
@@ -5,33 +5,42 @@ import Header from '../Header';
...
+      {hasData ? (
+        <div className={cc([classes['legend'], className])}>
+          <Header className={classes['legend-header']}>
+            <>
```
Comment (dvira) [inline src/components/common/WMLegend/index.tsx:30]: This fragment however is unnecessary.

File: src/components/common/buttons/AddButton/index.tsx [hunk 24]
```diff
@@ -0,0 +1,21 @@
...
+export default function AddButton({
+  className,
+  onClick,
+}: {
+  className?: string;
+  onClick?: () => void;
+}): ReactElement {
+  return (
+    <WMButton className={cc([classes['add-button'], className])} onClick={onClick}>
+      <PlusOutlined />
```
Comment (dvira) [inline src/components/common/buttons/AddButton/index.tsx:18]: Please replace this icon with our 'plus' icon.

File: src/components/common/buttons/AddButton/style.module.scss [hunk 25]
```diff
@@ -0,0 +1,20 @@
+.add-button {
+  &:global(.ant-btn) {
+    height: 28px;
+    width: 28px;
+    box-shadow: 0 2px 0 0 var(--shadow);
```
Comment (dvira) [inline src/components/common/buttons/AddButton/style.module.scss:5]: We already have a css var for box-shadows, please check before adding vars.

File: src/components/common/charts/PieBarSummary/index.tsx [hunk 26]
```diff
@@ -10,12 +10,18 @@ export default function PieBarSummary({
...
-      {text && <span className={classes.text}>{text}</span>}
+      {value ? (
+        <>
+          <span className={classes.value}>{value}</span>
+          {unit && <span className={classes.unit}>{unit}</span>}
+          {text && <span className={classes.text}>{text}</span>}
+        </>
+      ) : (
+        <span className={classes.value}>- - : - -</span>
+      )}
```
Comment (dvira) [inline src/components/common/charts/PieBarSummary/index.tsx:24]: I know it's not you, but please change all classes written here to this format: `classes['some-class']`. Even if its only one word, thanks.

File: src/components/common/charts/WMLineChart/index.tsx [hunk 27]
```diff
@@ -12,6 +12,7 @@ import {
 import { ITooltipContent } from '../charts.interface';
 import WMChartTooltip from '../WMChartTooltip';
```
Comment (dvira) [inline src/components/common/charts/WMLineChart/index.tsx:13]: Please add a line break.

File: src/components/common/charts/WMLineChart/emptyLineChart.tsx [hunk 28]
```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from 'react';
+import { LineChart, XAxis, YAxis, CartesianGrid, Line, ResponsiveContainer } from 'recharts';
+
+import emptyChartMockData from './emptyChartMock';
+
+import classes from './style.module.scss';
+
+export default function EmptyLineChart({ className }: { className?: string }): ReactElement {
```
Comment (dvira) [inline src/components/common/charts/WMLineChart/emptyLineChart.tsx:8]: Is 'className' prop really necessary?
Please capitalize this component's file name.

File: src/components/common/charts/WMLineChart/style.module.scss [hunk 29]
```diff
@@ -0,0 +1,3 @@
+.course-summary-chart {
+  height: 250px;
+}
```
Comment (dvira) [inline src/components/common/charts/WMLineChart/style.module.scss:3]: I saw you utilized the height prop in `EmptyLineChart`, is this file necessary?
If it's only for the height I would lose this file and just use inline styles in the 'empty' component.

File: src/styles/variables.scss [hunk 30]
```diff
@@ -9,6 +9,7 @@
   --white: #fff;
   --background: #fff;
   --selected: #d0e8ff;
+  --shadow: #f2f2f2;
```
Comment (dvira) [inline src/styles/variables.scss:12]: Already exists on line 6.

File: src/styles/variables.scss [hunk 31]
```diff
@@ -53,6 +54,8 @@
   --neutral200: #b3b3b3;
   --neutral100: #e1e9ee;
 
+  --empty-text: rgba(0, 0, 0, 0.5);
```
Comment (dvira) [inline src/styles/variables.scss:57]: Placing any css property with an empty line before or after should yield this stylelint warning:
`Unexpected empty line before custom property (custom-property-empty-line-before)stylelint(custom-property-empty-line-before)`.

File: src/components/Screen/CourseEditorScreen/CourseOutlineTab.tsx [hunk 32]
```diff
@@ -1,4 +1,5 @@
 import React, { ReactElement } from 'react';
+import cc from 'classcat';
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CourseEditorScreen/CourseOutlineTab.tsx:2]: Removed

File: src/components/Screen/CourseEditorScreen/ResourcesList.tsx [hunk 33]
```diff
@@ -66,7 +71,29 @@ export default function ResourcesList(): ReactElement {
           onSearch={onSearch}
         />
       </div>
-      <CourseItemsList items={getCourseItems(filteredCourseItems)} />
+      <div
+        className={cc([
+          classes['items-container'],
+          {
+            [classes['empty-state-items']]: !isFilteredCourseItems,
+          },
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CourseEditorScreen/ResourcesList.tsx:79]: Fixed it

File: src/components/Screen/CourseEditorScreen/ResourcesList.tsx [hunk 34]
```diff
@@ -66,7 +71,29 @@ export default function ResourcesList(): ReactElement {
...
+            description={
+              <>
+                <div>There are no items available.</div>
+                <div>Create Walk-thrus and resources from the WalkMe Editor.</div>
+              </>
+            }
+          />
+        ) : (
+          <WMEmpty description="No results found" />
+        )}
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CourseEditorScreen/ResourcesList.tsx:95]: Right, flipped it

File: src/components/Screen/CourseEditorScreen/ResourcesList.tsx [hunk 35]
```diff
@@ -48,6 +50,9 @@ export default function ResourcesList(): ReactElement {
     onSearch(courseItemsSearchValue);
   };
 
+  const isCourseItems = courseItems?.length === 0 ? false : true;
+  const isFilteredCourseItems = filteredCourseItems?.length === 0 ? false : true;
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CourseEditorScreen/ResourcesList.tsx:54]: I changed the name, let's talk about the first comment , i don't understand the problem.

File: src/components/Screen/CourseEditorScreen/style.module.scss [hunk 36]
```diff
@@ -52,6 +52,17 @@
...
+
+    .empty-state-items {
+      flex-grow: 1;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      .item-list {
+        margin: unset;
+      }
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CourseEditorScreen/style.module.scss:64]: Right

File: src/components/Screen/CourseEditorScreen/style.module.scss [hunk 37]
```diff
@@ -60,6 +71,7 @@
     .tabs {
       height: inherit;
       color: inherit;
+      height: 100%;
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CourseEditorScreen/style.module.scss:74]: OK

File: src/components/Screen/CoursesScreen/index.tsx [hunk 38]
```diff
@@ -114,6 +130,17 @@ export default function CoursesScreen(): ReactElement {
         title={`${tableData.length} ${CoursesTableTitle}`}
         subTitle="Courses will appear to your users in the order below. Drag & Drop items to change their order."
       >
+        <ConfigProvider renderEmpty={customizeRenderEmpty}>
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/Screen/CoursesScreen/index.tsx:133]: 'data={tableData as Array<ICourseData>}'
When 'tableData =[]' the empty state pops, i wrote it up, on the PR comments .

File: src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx [hunk 39]
```diff
@@ -47,15 +47,26 @@ export default function CourseSummaryChart({ summaryData }: ICourseSummaryChart)
...
   return (
     <WMCard title={title}>
       <div className={classes['course-summary']}>
         <div className={classes['chart-legend']}>
-          <WMLegend title="User Started" dotStatusColor="#F2B529">
-            <LegendContent number={2580} description="52% of users with TeachMe access" />
+          <WMLegend title="Users Started" dotStatusColor="#F2B529" hasData={hasData}>
+            <LegendContent
+              number={usersNumber('Users Started')}
+              description="52% of users with TeachMe access" //TODO: calc %
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx:62]: Ok

File: src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx [hunk 40]
```diff
@@ -47,15 +47,26 @@ export default function CourseSummaryChart({ summaryData }: ICourseSummaryChart)
     });
   };
 
+  const hasData = Boolean(summaryData.data.days.length);
+
+  const usersNumber = (key: string): number =>
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/common/AnalyticsCharts/CourseSummaryChart/index.tsx:52]: OK

File: src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx [hunk 41]
```diff
@@ -13,21 +13,24 @@ export default function QuizScoreChart({ quizScoreData }: IQuizScoreData): React
     data: { average = 0, passmark = 0 },
   } = quizScoreData;
 
+
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx:16]: Done

File: src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx [hunk 42]
```diff
@@ -13,21 +13,24 @@ export default function QuizScoreChart({ quizScoreData }: IQuizScoreData): React
...
   return (
     <WMCard title={title} className={classes['course-average']}>
       <div className={classes['course-average-content']}>
         <WMProgress
           className={classes['course-average-chart']}
           percent={average}
           type={ProgressType.Circle}
-          format={() => average}
+          format={quizScoreData.data ? () => average : () => '- -'}
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx:24]: Fixed it

File: src/components/common/Icon/icon.interface.ts [hunk 43]
```diff
@@ -20,5 +20,7 @@ export enum IconType {
   LessonSmall = 'lesson-small',
   Refresh = 'refresh',
   Plus = 'plus',
+  CourseEmpty = 'course-empty',
+  LessonEmpty = 'lesson-empty',
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/common/Icon/icon.interface.ts:24]: Changed it

File: src/components/common/WMEmpty/index.tsx (hunk 14 above)
Comment (shiranmeir) [reply src/components/common/WMEmpty/index.tsx:10]: Ok

File: src/components/common/WMLegend/EmptyLegend.tsx (hunk 15 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/EmptyLegend.tsx:11]: Ok

File: src/components/common/WMLegend/EmptyLegend.tsx (hunk 16 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/EmptyLegend.tsx:14]: Right

File: src/components/common/WMLegend/EmptyLegend.tsx (hunk 17 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/EmptyLegend.tsx:15]: Ok

File: src/components/common/WMLegend/EmptyLegend.tsx (hunk 18 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/EmptyLegend.tsx:17]: Ok

File: src/components/common/WMLegend/EmptyLegend.tsx (hunk 19 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/EmptyLegend.tsx:23]: Removed it

File: src/components/common/WMLegend/index.tsx (hunk 20 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/index.tsx:8]: OK

File: src/components/common/WMLegend/index.tsx (hunk 21 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/index.tsx:24]: Ok

File: src/components/common/WMLegend/index.tsx (hunk 22 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/index.tsx:26]: Removed it

File: src/components/common/WMLegend/index.tsx (hunk 23 above)
Comment (shiranmeir) [reply src/components/common/WMLegend/index.tsx:30]: Removed it

File: src/components/common/buttons/AddButton/index.tsx (hunk 24 above)
Comment (shiranmeir) [reply src/components/common/buttons/AddButton/index.tsx:18]: Done

File: src/components/common/buttons/AddButton/style.module.scss (hunk 25 above)
Comment (shiranmeir) [reply src/components/common/buttons/AddButton/style.module.scss:5]: Ok

File: src/components/common/charts/PieBarSummary/index.tsx (hunk 26 above)
Comment (shiranmeir) [reply src/components/common/charts/PieBarSummary/index.tsx:24]: Ok done

File: src/components/common/charts/WMLineChart/index.tsx (hunk 27 above)
Comment (shiranmeir) [reply src/components/common/charts/WMLineChart/index.tsx:13]: Ok

File: src/components/common/charts/WMLineChart/emptyLineChart.tsx (hunk 28 above)
Comment (shiranmeir) [reply src/components/common/charts/WMLineChart/emptyLineChart.tsx:8]: No
Done

File: src/components/common/charts/WMLineChart/style.module.scss (hunk 29 above)
Comment (shiranmeir) [reply src/components/common/charts/WMLineChart/style.module.scss:3]: Ok Removed

File: src/styles/variables.scss (hunk 30 above)
Comment (shiranmeir) [reply src/styles/variables.scss:12]: Removed

File: src/styles/variables.scss (hunk 31 above)
Comment (shiranmeir) [reply src/styles/variables.scss:57]: Ok

File: src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx [hunk 44]
```diff
@@ -13,21 +13,24 @@ export default function QuizScoreChart({ quizScoreData }: IQuizScoreData): React
...
           width={80}
           strokeWidth={10}
           status={average > passmark ? ProgressStatus.Success : ProgressStatus.Exception}
         />
-        <span className={classes['passmark']}>
-          Passmark: <b>{passmark}</b>
-        </span>
+        {quizScoreData.data && (
```
[the remark this answers is not in the document]
Comment (shiranmeir) [reply src/components/common/AnalyticsCharts/QuizScoreChart/index.tsx:29]: Fixed it
